spec/collator.html

```diff
@@ -49,46 +49,39 @@
 
       <emu-alg>
         1. If collator has an [[initializedIntlObject]] internal slot with value *true*, throw a *TypeError* exception.
-        2. Set collator.[[initializedIntlObject]] to *true*.
-        3. Let _requestedLocales_ be CanonicalizeLocaleList(_locales_).
-        4. ReturnIfAbrupt(_requestedLocales_).
-        5. If _options_ is *undefined*, then
+        1. Set collator.[[initializedIntlObject]] to *true*.
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
           1. Let _options_ be ObjectCreate(*%ObjectPrototype%*).
-        6. Else
-          1. Let _options_ be ToObject(_options_).
-          1. ReturnIfAbrupt(_options_).
-        7. Let _u_ be GetOption(_options_, *"usage"*, *"string"*, «*"sort"*, *"search"*», *"sort"*).
-        8. ReturnIfAbrupt(_u_).
-        9. Set _collator_.[[usage]] to _u_.
-        10. If _u_ is *"sort"*, then
-          1. Let _localeData_ be the value of *%Collator%*.[[sortLocaleData]];
-        11. Else
+        1. Else
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _u_ be ? GetOption(_options_, *"usage"*, *"string"*, «*"sort"*, *"search"*», *"sort"*).
+        1. Set _collator_.[[usage]] to _u_.
+        1. If _u_ is *"sort"*, then
+          1. Let _localeData_ be the value of *%Collator%*.[[sortLocaleData]].
+        1. Else,
```

Comment:
Right. It looks like @bterlson is using these two conventions: 

- comma: 

  ```
  1. Else,
    1. When there is
    2. More than one line
  ```
- no comma:

  ```
  1. Else if there is only one line
  ```